versions/1.0/parsers/antlr4/WdlLexer.g4

```diff
@@ -53,6 +56,7 @@ BoolLiteral
 	: 'true'
 	| 'false'
 	;
+Null: 'null';
```

Comment:
`Null` did not make its way into the grammar until the upcoming development version. IN v1.0 and draft-2 it is not valid

Reply:
It appears in the 1.0 spec https://github.com/openwdl/wdl/blob/master/versions/1.0/SPEC.md#parameter-metadata-section

Reply:
ugh, this is a sneaky ugly addition. Adding a `null` keyword that can only be used in a given parameter meta block. 

`null` is not a valid special  value anywhere else except here. I would be in favor of not treating it special at the level of the parser/lexer, but allow the implementations to worry about this. The reason being is that the following is valid under v1.0 since `null` is not technically a keyword, except in the case of the meta
```wdl
workflow a {
 input {
   String null
 }
}
```
Inn this case `null` SHOULD evaluate to whatever the user set it as. However, if we add a lexer rule for this, `null` will not be considererd an `identifer` and this workflow would fail to parse.

I say we treat `null` as a special case wihtin the downstream libraries. IN the general case its considered an identifier, however within the `parameter-meta` section, if the `null` value is not set, then the library should evaluate this to a literal `null`

Reply:
Gotcha, that makes sense. I'll revert.

Reply:
But then shouldn't meta_kv be `Identifier COLON expr?`?

Reply:
Oh nevermind, I see what you're saying - treat null as an identifier in the parser, but the engine should special-case the null identifier - if it doesn't match any declaration then treat it as null literal.